about.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="shortcut icon"
+      href="./Images/service_camp.jpeg"
+      type="image/x-icon"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>About Page</title>
+  </head>
+  <body>
+    <main>
+      <nav class="social">
+        <ul class="social-container">
+          <li class="social-item"><a href=""></a></li>
+          <li class="social-item"
+            ><a href=""><i class="fa-brands fa-facebook-square"></i></a
+          ></li>
+          <li class="social-item"><a href="">logout</a></li>
+        </ul>
+      </nav>
+      <nav class="navbar">
+        <div class="menu">
+          <a href="./index.html"
+            ><img src="./Images/nomiddlemen.png" alt=""
+          /></a>
+          <div class="hamburger"></div>
+        </div>
+        <div>
+          <ul class="list-container">
+            <li class="list-item"><a href="./index.html">Home</a></li>
+            <li class="list-item"><a href="">See Product</a></li>
+            <li class="list-item"><a href="">Signup As A Talent</a></li>
+            <li class="list-item"><a href="">Sponsor</a></li>
+            <li class="list-item"><a href=""></a></li>
+          </ul>
+        </div>
+      </nav>
+      <section class="section-about">
+        <header class="header-about">
+          <p>Service Camp!</p>
+          <h1
+            >BRINGING SKILLS AND PRODUCTS TO YOUR DOORSTEP, NO MORE
+            MIDDLEMEN</h1
+          >
+        </header>
+        <article class="article-1">
+          <p
+            >No middle man, after searching for solutions we have built a
+            searchable directory of products and brands that are truly direct to
+            customers. We offer customers sustainable, valuable products through
+            low prices, high product quality, and outstanding customer
+            service.</p
+          >
+        </article>
+        <p class="reach"
+          >Please Reach out to us at: <span>servicecamp001@gmail.com</span>
+          <br />
+          If you would like to suggest a consumer brand for our directory</p
+        >
+      </section>
+      <section class="logo">
+        <header>
+          <h2>Service camp Goal and Logo</h2>
+          <span></span>
+        </header>
```

Comment:
- [ ] To improve UI and UX of the project, I recommend decreasing the size of the image here, because it is taking too much if users attention:
 
Behance design  | Your design
------------- | -------------
<img width="780" alt="Screenshot 2022-04-15 at 21 11 18" src="https://user-images.githubusercontent.com/77038610/163600259-5a2f4c51-d786-40e0-bf40-23f6a9463f7a.png">  | <img width="769" alt="Screenshot 2022-04-15 at 21 10 32" src="https://user-images.githubusercontent.com/77038610/163600179-46c241d0-d01a-4042-9319-8c76163e7a8f.png">

Reply:
fixed